app/components/Education/Education.scss

```
@@ -1,41 +1,34 @@
+  font-weight: 700;
+  margin-bottom: 1rem;
 }
 
 .education-link {
   @include link;
 }
+
+@media print {
+  .education-item {
+    break-inside: avoid;
+    page-break-inside: avoid;
```

Stylelint (17.6.0)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)